task8/src/main/java/AdsServlet.java

```diff
@@ -0,0 +1,57 @@
+import adList.Ad;
+import adList.AdList;
+import com.google.gson.Gson;
+
+import javax.servlet.*;
+import javax.servlet.http.*;
+import javax.servlet.annotation.*;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+@WebServlet(name = "AdsServlet", value = "/ads")
+public class AdsServlet extends HttpServlet {
+    AdList adList;
```

Comment:
Make internal fields of a class `private`. If outside access is necessary, create getters for them